core/collector.class.inc.php

```diff
@@ -69,6 +74,15 @@ public function __construct() {
 		$this->aSkippedAttributes = [];
 	}
 
+	public static function SetCallItopService($oCurrentCallItopService=null){
+		if (null === $oCurrentCallItopService){
+			$oCurrentCallItopService = new CallItopService();
+		}
+
+		Collector::$oCallItopService = $oCurrentCallItopService;
```

Comment:
```suggestion
		static::$oCallItopService = $oCurrentCallItopService;
```

Reply:
+1

Reply:
+1

Reply:
Not resolved..

Reply:
This suggestion is missing from the applied ones.